### ISSUE TITLE
Allow more than 4 registers for 74HC165 and 74HC595

### ISSUE THREAD

**Issue by srobert72** [MobiFlight/MobiFlight-Connector#1323]
Hello,

**Is your feature request related to a problem? Please describe.**
Input and ouput registers are limited to a maximum of 4 chips in MobiFlight.
But technically, 74HC165 and 74HC595 registers allow a maximum of 8 chips in a chain.
Could it be possible to address 8 chips also in MobiFlight ?

**Describe the solution you'd like**
Just increase the dropdown selector up to a maximum of 8 items instead of 4.

**Describe alternatives you've considered**
There is no way to use more than 4 shift registers in MobiFlight.

Thank you
Regards
Stephane

**Comment by DocMoebiuz** [comment]
@elral - with no limit on the FW side, this now is totally limited by the boards.json only

**Comment by elral** [comment]
@DocMoebiuz Yes, but the user is requesting 8 shifters **daisy chained**. No changes would be required in the firmware for this and in the board.json file.

**Comment by srobert72** [comment]
Is it only limited by the dropdown selector of the user interface in win10 application ?
Easy to change ?

**Comment by srobert72** [comment]
I tried to create a branch and push code to make a PR, but I've got 403 error.

So I give you my changes, just 2 lines of code:
for both
https://github.com/MobiFlight/MobiFlight-Connector/blob/main/UI/Panels/Device/MFShiftRegisterPanel.cs
https://github.com/MobiFlight/MobiFlight-Connector/blob/main/UI/Panels/Device/MFInputShiftRegisterPanel.cs

set
`private int MAX_MODULES = 8;`
instead of
`private int MAX_MODULES = 4;`

Here is a patch:
[Fix_ShiftRegister_and_InputShiftRegister_chips_up_to_MAX_MODULES=8.patch](https://github.com/MobiFlight/MobiFlight-Connector/files/13341390/Fix_ShiftRegister_and_InputShiftRegister_chips_up_to_MAX_MODULES.8.patch)

**Comment by DocMoebiuz** [comment]
We have removed the device limitations in the firmware. These come now from the settings in the boards.json file. Please experiment with it.

This was the PR on the firmware side: https://github.com/MobiFlight/MobiFlight-FirmwareSource/pull/253

It is already included in the beta. And it will be included in the next release.

I will close this issue.

**Comment by DocMoebiuz** [comment]
Sorry, my bad - you were looking at the number of modules in the chain.

**Comment by srobert72** [comment]
Hello @DocMoebiuz,
Is there any chance this ticket will be fixed?
Thank you
Regards
Stephane
